src/argus/notificationprofile/models.py

```diff
@@ -140,17 +142,20 @@ def is_empty(self):
             and self.is_event_type_empty()
         )
 
-    def incident_fits_tristates(self, incident):
+    def incident_fits_tristates(self, incident) -> Dict[str, TriState]:
```

Comment:
How about just explicitly:
```suggestion
    def get_incident_tristate_checks(self,` incident) -> Dict[str, TriState]:
    """Returns a dict containing the incident fitting results of each tristate filter"""
```